### ISSUE TITLE
Support ECW imagery format for GEE Fusion source data import

### ISSUE THREAD

**Issue by avnishb** [google/earthenterprise#257]
Build GDAL with support for ECW source files:

http://www.gdal.org/frmt_ecw.html

**Comment by bradh** [comment]
This will be tough to do in a nice open source way. Perhaps relying on native GDAL would be the sane approach here, as for MrSID and JPEG2000.

**Comment by avnishb** [comment]
Agreed, especially since the (proprietary) ECW SDK is currently only available as a binary-- which can complicate the build process.  But if users have a need for this imagery format and have licensed the SDK, it would be useful to provide support for it.

P.S. I think there's an old & unsupported version of the ECW source code floating around that is under a GPL license.

**Comment by bradh** [comment]
ECW SDK 3.x allowed writing of small files, but I don't think it was under GPL...
